db/item_db.conf

```diff
@@ -150872,6 +150872,27 @@ item_db: (
 
 //== New Etc Items =========================================
 {
+	Id: 22508
+	AegisName: "Para_Team_Mark_"
+	Name: "Eden Group Mark"
+	Type: 11
+	Buy: 0
+	Delay: 1200000
+	Trade: {
+		nodrop: true
+		notrade: true
+		noselltonpc: true
+		nocart: true
+		nostorage: true
+		nogstorage: true
+		nomail: true
+		noauction: true
+	}
+	Script: <"
+		unitskilluseid getcharid(3),"AL_TELEPORT",3;
```

Comment:
Is there any problem with this pull? Please, tell me to fix it. This is an official feature. With the information I have, kRO has the use marked in this pull (teleport to save point) and then change it to teleport to paradise group second floor. Both are official behavior, but, one is outdated.

Until having the new behavior, would it be bad to apply the last official behavior? The only thing that may be wrong is that Cool Down was 30 minutes when teleporting to save point, then was 20 minutes when teleporting to second floor of paradise group. But both are official (also in kRO).

If do you need any confirmation for official dialogues, we still have some eden group mark on iRO to convert to the new one.

Any answer would be appreciated.